### DISCUSSION TITLE
"Start All" via CLI

### DISCUSSION THREAD

**Discussion by Uldrix** [leokhoa/laragon#444] [General]
Hello,
For some reasons i need to embed a portable version of laragon into an electronjs application.
Actualy what the button "start all" performs, i want to do it from a command line silently without using the laragon.exe, and without installing as service/autostart (no admin rights).
I guess there are some exe to run in the bin folder in some order. I couldnt find this in the documentation.
Any help would be great.
Thanks.

**Comment by Uldrix** [comment]
Finaly i figured it out, it's something like this for the portable version : 

```shell
cd laragon-6.0.0
bin\nginx\nginx-1.14.0\nginx -p "bin\nginx\nginx-1.14.0"
bin\mysql\mysql-5.1.72-win32\bin\mysqld  --log-error=bin\mysql\mysql-5.1.72-win32\data\mysqld.log
bin\php\php-5.4.9-nts-Win32-VC9-x86\php-cgi.exe -b localhost:9003
bin\php\php-5.4.9-nts-Win32-VC9-x86\php-cgi.exe -b localhost:9004 
```

**Reply by Pen-y-Fan** [reply]
There are cli commands for Laragon too: [laragon > docs > Command line](https://laragon.org/docs/cli.html)

**Reply by Pen-y-Fan** [reply]
If you install Laragon as an administrator you may need to change the permission to run under a standard user account, see my blog: [How to set up Laragon on a new Windows computer (part 2) - permissions](https://pen-y-fan.github.io/2022/11/08/how-to-set-up-laragon-on-a-new-windows-computer-part-2/)

**Reply by Uldrix** [reply]
yeah this link you are showing just show how to reload an already started nginx/apache/etc. via laragon. It is not what i wanted.
Also where i need to use it, there is no admin rights.
And the users who i will set it for will not deal with the laragon panel.
So i will embed this, and start nginx/mysql/php via electronjs process in a seamless way. What i was missing was how to do it but i figured it out by tracing laragon with process explorer and got the exact command lines and order of how to do it :-)
I need to do this for a small project, where laragon is the perfect choice and will prevent me wasting months of displomatic negociations to get a corporate server, that i might not obtain in the endline...

**Reply by brahimjid** [reply]
> yeah this link you are showing just show how to reload an already started nginx/apache/etc. via laragon. It is not what i wanted. Also where i need to use it, there is no admin rights. And the users who i will set it for will not deal with the laragon panel. So i will embed this, and start nginx/mysql/php via electronjs process in a seamless way. What i was missing was how to do it but i figured it out by tracing laragon with process explorer and got the exact command lines and order of how to do it :-) I need to do this for a small project, where laragon is the perfect choice and will prevent me wasting months of displomatic negociations to get a corporate server, that i might not obtain in the endline...

can you show me the way
